### PR TITLE
Update German Localization

### PR DESCRIPTION
**Pull request by DaCoolX**

### CODE REVIEW

File: resources/l10n/de/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -805,87 +813,95 @@ msgstr "Hinzufügen/Entfernen"
...
 
-#: src/gui/qt/settings/general_tab.cpp:151
+#: src/gui/qt/settings/general_tab.cpp:158
 msgid "Masterlist prelude source"
 msgstr "Masterlist Prelude Quelle"
 
-#: src/gui/qt/settings/general_tab.cpp:154
+#: src/gui/qt/settings/general_tab.cpp:160
+msgid "Display dialog when sorting makes no changes"
+msgstr "Zeige Dialog wenn Sortieren keine Änderungen macht"
```
Comment (DaCoolX) [inline resources/l10n/de/LC_MESSAGES/loot.po:889]: @pStyl3 Does this make sense for where it is used?

Comment (pStyl3) [reply resources/l10n/de/LC_MESSAGES/loot.po:889]: I think your translation is okay. See [here](https://github.com/loot/loot/issues/1729#issuecomment-1106704528) for more information on the subject.

File: resources/l10n/de/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -21,79 +21,87 @@ msgstr ""
...
 
-#: src/gui/qt/filters_widget.cpp:208
+#: src/gui/qt/filters_widget.cpp:248
 msgid "Hide Creation Club plugins"
 msgstr "Creation Club Plugins ausblenden"
 
-#: src/gui/qt/filters_widget.cpp:209
+#: src/gui/qt/filters_widget.cpp:249
+msgid "Show only empty plugins"
+msgstr "Zeige nur leer Plugins"
```
Comment (pStyl3) [inline resources/l10n/de/LC_MESSAGES/loot.po:78]: It should be `leere`.

File: resources/l10n/de/LC_MESSAGES/loot.po [hunk 3]
```diff
@@ -651,32 +655,36 @@ msgstr "Finde Vorheriges"
...
 
-#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:162
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:172
 msgid "Group already exists!"
 msgstr "Gruppe existiert bereits!"
 
-#: src/gui/qt/plugin_editor/group_tab.cpp:92
+#: src/gui/qt/plugin_editor/group_tab.cpp:56
+msgid "%1 (Group does not exist)"
+msgstr "%1% (Gruppe existiert nicht)"
```
Comment (pStyl3) [inline resources/l10n/de/LC_MESSAGES/loot.po:685]: The second `%` isn't needed, `%1` is sufficient.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

File: resources/l10n/de/LC_MESSAGES/loot.po (hunk 2 above)
Comment (DaCoolX) [reply resources/l10n/de/LC_MESSAGES/loot.po]: Done

File: resources/l10n/de/LC_MESSAGES/loot.po (hunk 3 above)
Comment (DaCoolX) [reply resources/l10n/de/LC_MESSAGES/loot.po]: Done

**Review by pStyl3 — APPROVED** [review]

**Comment by Ortham** [comment]
Thanks! 👍

**Pull request merged**